autopush/web/base.py

```diff
@@ -144,6 +113,7 @@ def _response_err(self, fail):
 
         """
         fmt = fail.value.message or 'Exception'
+        fail._client_info = self._client_info
```

Comment:
Ok, so the _client_info is attached to the failure object itself.... but above we search the event for 'client_info'. This works?